quesma/clickhouse/schema_loader.go

```diff
@@ -92,31 +96,55 @@ func (sl *tableDiscovery) ReloadTableDefinitions() {
 		sl.tableDefinitions.Store(NewTableMap())
 		return
 	} else {
-		for table, columns := range tables {
-			if indexConfig, found := sl.cfg.IndexConfig[table]; found {
-				if indexConfig.Enabled {
-					for colName := range columns {
-						if _, exists := indexConfig.Aliases[colName]; exists {
-							logger.Error().Msgf("column [%s] clashes with an existing alias, table [%s]", colName, table)
+		if sl.TableAutodiscoveryEnabled() {
+			var autoDiscoResults strings.Builder
+			logger.Info().Msg("Index configuration empty, running table auto-discovery")
+			for table, columns := range tables {
+				comment := sl.SchemaManagement.tableComment(databaseName, table)
+				createTableQuery := sl.SchemaManagement.createTableQuery(databaseName, table)
+				var maybeTimestampField string
+				if sl.cfg.Hydrolix.IsNonEmpty() {
+					maybeTimestampField = sl.SchemaManagement.tableTimestampField(databaseName, table, "hydrolix")
```

Comment:
I think that these strings `hydrolix` and `clickhouse` could be extracted as consts and used also in `tableTimestampField ` function

Reply:
https://github.com/QuesmaOrg/quesma/pull/497/commits/c61928a18c8ba4d8398b4d8158299c9d229776fe